grpc.go

```diff
@@ -524,6 +525,76 @@ func (a *agentGRPC) rollbackFailingContainerCreation(ctr *container) {
 	}
 }
 
+func changeToBundlePath(spec *specs.Spec) (string, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return cwd, err
+	}
+
+	rootfsPath := spec.Root.Path
```

Comment:
I know that `spec` and `spec.Root` and `spec.Root.Path` MUST be specified, but I'd still opt for paranoia and check they are not `nil`, not `nil` and `!= ""` since if this resolves to `""`, the function will do nothing and the `config.json` will be written to `cwd` I think?